junction/static/less/app.less

```diff
@@ -0,0 +1,10 @@
+@import "vendor.less";
+
+@import "variables.less";
+@import "bootstrap_overrides.less";
+
+@import "utils.less";
+@import "base.less";
+
+@import "editor.less";
+@import "home.less";
```

Comment:
Awesomeness! Wasn't aware `less` has mixins. Then I think the feature difference is really not much of the debate on top of the awesome work you have already done. Let me know when you are done with changes required for this PR and we are good to get this in!! :blush: